conda/recipes/cudf/meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
     - setuptools
     - numba >=0.49.0
     - dlpack
-    - pyarrow 0.15.0.*
+    - pyarrow 0.17.1.*
```

I think we can relax this as the pinning will be determined at the `libcudf`/`arrow-cpp` level. So this will just fall out correctly.

```suggestion
    - pyarrow
```